app/models/supplier.rb

```diff
@@ -51,11 +53,11 @@ def sync_all
     end
     # Find any new articles, unless the import is manual
     unless shared_sync_method == 'import'
-      for shared_article in shared_supplier.shared_articles
-        unless articles.undeleted.find_by_order_number(shared_article.number)
-          new_articles << shared_article.build_new_article(self)
-        end
-      end
+      shared_supplier
+        .shared_articles
+        .where(available: true)
```

Comment:
It looks like the `available` attribute is something you have in your sharedlists, but which isn't present in the [stock version](https://github.com/foodcoops/sharedlists/blob/master/db/schema.rb#L16), right? 
This [came up on the mailing-list](http://foodsoft.51229.x6.nabble.com/Configuring-sharedlists-for-Terra-tp2076p2096.html). I'll remove the `.where(available: true)`.

Reply:
i suppose i must have @wvengen - it has been many years, but it doesn't appear to exist in the original queries.  my apologies for the support hassle.

fwiw, i built my own shared-list app in node.js - it is a horrendous regex monster, but it usually gets 99% of the information out of a produce catalog string.

Reply:
No probs, we missed this during review. I just corrected it on master.
Interesting, feel free to add a link to the wiki (on a new "Related software" page or so).